.travis.yml

```diff
@@ -19,3 +19,9 @@ deploy:
   local_dir: build
   on:
     branch: master
+os:
+  - windows
```

Comment:
I restarted the Travis CI job for the windows build, it looks like it stalled out.